shared/teams/render.js.flow

```diff
@@ -0,0 +1,10 @@
+// @flow
```

Comment:
We'll eventually have a teams dialog on mobile right? So I should keep the types into their own .flow file even though it's desktop-only now.

Reply:
We should likely start with unified views and split them if we need to. You can peek at the mobile designs and see if their different

Reply:
Done, added button (behind a flag) on mobile and tweaked stuff to make it look ok on mobile.